lib/SIL/Verifier/LinearLifetimeChecker.cpp

```diff
@@ -245,6 +259,53 @@ void State::initializeAllNonConsumingUses(
 //                        Consuming Use Initialization
 //===----------------------------------------------------------------------===//
 
+void State::initializeAllExtendLifetimeUses(
+    ArrayRef<Operand *> linearLifetimeEndingUses,
```

Comment:
Could this parameter be `extendLifetimeUses` to make things clear?

Reply:
Oops.  Fixed.